bin/prometheus-metadata-exporter

```diff
@@ -113,16 +113,27 @@ class OcspCheckMetric < Prometheus::Client::Gauge
   private
 
   def check_and_set(gauge, certificate_identities, ca_files)
+      return if certificate_identities.nil?
+
       pems = certificate_identities.keys
       ocsp_results = @pem_checker.check_pems(pems, ca_files)
       certificate_identities.map do |pem, identities|
         identities.map do |identity|
+          puts(identity)
```

Comment:
Is this debug logging, and should it stay in?

Reply:
Oops yes il remove that.